src/main/java/frc/robot/subsystems/ramp/RampSubsystem.java

```diff
@@ -30,4 +30,9 @@ public void prepareForIntake() {
   public boolean isInPosition() {
     return mechanism.inPosition();
   }
+
+  public boolean isInClimbPosition() {
+    return mechanism.position == JsonConstants.rampConstants.climbPosition
```

Comment:
This should already be handled by inPosition above. Check to see if it is.


(Side Note)
In general we can't compare doubles with ==. So to do this type of check we need to add a tolerance here:

```
Math.abs(mechanism.position - JsonConstants.rampConstants.climbPosition) < JsonConstants.rampConstants.climbPositionTolerance
```

This should be how inPosition does it already